samples/NativeLibrary/NativeLibraryHost.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
```

Comment:
Here's how this looks:

![image](https://github.com/dotnet/runtimelab/assets/62474226/d0b864dd-09b4-4b37-85c7-3a6f9a460c54)